src/plugin/indexer-kafka/src/java/org/apache/nutch/indexwriter/kafka/package-info.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Would be good to have at least one sentence describing the indexer-kafka plugin, it will be shown on the [API docs overview page](https://builds.apache.org/job/nutch-trunk/javadoc/).

Reply:
Thanks, @AyalCiobotaru for taking the work to port #372 to master. Looks good. I'd need some time to test it.